CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ which uses the format described in
 to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][unreleased]
+### Added
+- Added environment variables `PLEDGE_MAILINGLIST_URL`
+  and `PLEDGE_MAILINGLIST_PRIVACY_NAME`.
 
```

Comment:
I didn't wanna mention about the Pledge CTA here yet.  Since this PR is about fixing env vars.